source/security/encryption/minio-kes.rst

```diff
@@ -0,0 +1,82 @@
+.. _minio-kes:
+
+============================
+MinIO Key Encryption Service
+============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+The MinIO Key Encryption Service (KES) is a stateless and distributed
+key-management system for high-performance applications. KES provides
+a bridge between applications running in bare-metal or orchestrated
+environments to centralised KMS solutions. 
+
+<DIAGRAM>
+
+KES is designed for simplicity, scalability, and security. It requires 
+minimal configuration to enable full functionality and requires only
+basic familiarity with cryptography or key-management concepts.
+
+MinIO servers require KES for performing Server-Side Encryption (SSE) of objects
+using Key Management Services (KMS). 
+
+KES Server Process
+------------------
+
+.. mc:: kes server
+
+:mc:`kes server` command starts the KES server. The :mc:`kes server` process
+handles requests for creating and retrieving cryptography keys from a supported
+Key Management System (KMS). 
+
+The command has the following syntax:
+
+.. code-block:: shell
+   :class: copyable
+
+   kes server --cert CERTIFICATE --key PRIVATEKEY --root ROOT_IDENTITY [OPTIONAL_FLAGS]
+
+:mc:`kes server` supports the following arguments:
+
+.. mc-cmd:: cert
+   :option:
+
+   *Required*
```

Comment:
KES does not have mandatory CLI flags. All are optional - and only required if the corresponding field in the config file is omitted.

Reply:
Done